### PR TITLE
Added new DADA2 reference db: sbdi-gtdb

### PR DESCRIPTION
**Pull request by erikrikarddaniel**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](<https://github.com/>nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint .`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

We have curated the GTDB 16S database with Sativa. This is now available as `sbdi-gtdb` with `--dada_ref_databases`.

### CODE REVIEW

**Review by d4straub — APPROVED** [review]
LGTM, but the failing linting needs to be solved.

> bin/taxref_reformat_sbdi-gtdb.sh:
	5: Trailing whitespace
	10: Wrong amount of left-padding spaces(want multiple of 4)
	12: Wrong amount of left-padding spaces(want multiple of 4)
	14: Wrong amount of left-padding spaces(want multiple of 4)
	16: Wrong amount of left-padding spaces(want multiple of 4)

File: bin/taxref_reformat_sbdi-gtdb.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+# Formatting script for sbdi-gtdb files.
+
+# We get the files with numbers as names. Loop over them and find out which
```
Comment (d4straub) [inline bin/taxref_reformat_sbdi-gtdb.sh:5]:
```suggestion
# We get the files with numbers as names. Loop over them and find out which
```

File: bin/taxref_reformat_sbdi-gtdb.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+# names.
+
+for f in *; do
+  if [ $(file -L $f | grep -c 'gzip') ]; then
+    if [ $(gunzip -c $f | head -n 1 | grep -c '>.*;.*;.*;.*;.*') -eq 1 ]; then
+      gunzip -c $f > assignTaxonomy.fna
+    elif [ $(gunzip -c $f | head -n 1 | grep -c '>.* .*') -eq 1 ]; then
+      gunzip -c $f > addSpecies.fna
+    fi
+  fi
```
Comment (d4straub) [inline bin/taxref_reformat_sbdi-gtdb.sh:16]:
```suggestion
for f in *; do
    if [ $(file -L $f | grep -c 'gzip') ]; then
        if [ $(gunzip -c $f | head -n 1 | grep -c '>.*;.*;.*;.*;.*') -eq 1 ]; then
            gunzip -c $f > assignTaxonomy.fna
        elif [ $(gunzip -c $f | head -n 1 | grep -c '>.* .*') -eq 1 ]; then
            gunzip -c $f > addSpecies.fna
        fi
    fi
```

**Review by d4straub — CHANGES_REQUESTED** [review]

File: bin/taxref_reformat_sbdi-gtdb.sh (hunk 2 above)
Comment (d4straub) [reply bin/taxref_reformat_sbdi-gtdb.sh:16]: I think that should solve it, a little hard to do that in the rudimentary github editor.

Comment (erikrikarddaniel) [reply bin/taxref_reformat_sbdi-gtdb.sh]: Sorry that you had to do that. I didn't notice until I had committed an update myself :-S

Comment (d4straub) [reply bin/taxref_reformat_sbdi-gtdb.sh]: No problem!

**Review by d4straub — APPROVED** [review]
LGTM!

**Pull request merged**